lib/static/new-ui/features/suites/components/TestSteps/index.tsx

```diff
@@ -52,13 +54,21 @@ function ListItemCorrupted({items, itemId}: TestStepProps): ReactNode {
     />;
 }
 
-function TestStep({onItemClick, items, itemId}: TestStepPropsActionable): ReactNode {
+function TestStep({onItemClick, items, itemId, index, onItemMouseMove}: TestStepPropsActionable): ReactNode {
     const item = items.structure.itemsById[itemId];
 
     if (item.type === StepType.Action) {
+        let nextAction: Action | null = null;
+        for (let i = index + 1; i < items.structure.visibleFlattenIds.length; i++) {
+            const maybeAction = items.structure.itemsById[items.structure.visibleFlattenIds[i]];
+            if (maybeAction.type === StepType.Action) {
```

Comment:
is it double check of the `StepType.Action` type?

Reply:
It's not a double-check. Here, we are trying to find next action in the list. Since the next item may not be an action but an attachment or something else, we have this check in place.